src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Attachments/index.js

```diff
@@ -1,33 +1,71 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {
+  Button,
+  themeColor,
+  themeSpacing,
+} from '@datapunt/asc-ui';
 
 import { attachmentsType } from 'shared/types';
 
-import './style.scss';
+const StyledDL = styled.dl`
+  margin: 0;
+`;
+
+const StyledDefinition = styled.dt`
+  font-weight: normal;
+  width: 25%;
+  display: inline-block;
+  color: ${themeColor('tint', 'level5')};
+
+  margin: 0;
+  vertical-align: top;
+  padding: ${themeSpacing(2)} 0;
+`;
+
+const StyledValue = styled.dd`
+  width: 75%;
+  display: inline-block;
+
+  margin: 0;
+  vertical-align: top;
+  padding: ${themeSpacing(2)} 0;
+`;
+
+const StyledButton = styled(Button)`
+  display: inline-block;
```

Comment:
`cursor: pointer` Is missing. Isn't it necessary?

Reply:
no it is gone because the <Button> component take care of that